plugins/source/aws/docs/tables/aws_cognito_user_pool_identity_providers.md

```diff
@@ -14,16 +14,16 @@ This table depends on [aws_cognito_user_pools](aws_cognito_user_pools.md).
 
 | Name          | Type          |
 | ------------- | ------------- |
-|_cq_id (PK)|`uuid`|
+|_cq_id|`uuid`|
 |_cq_parent_id|`uuid`|
 |account_id|`utf8`|
 |region|`utf8`|
-|user_pool_arn|`utf8`|
+|user_pool_arn (PK)|`utf8`|
 |attribute_mapping|`json`|
 |creation_date|`timestamp[us, tz=UTC]`|
 |idp_identifiers|`list<item: utf8, nullable>`|
 |last_modified_date|`timestamp[us, tz=UTC]`|
 |provider_details|`json`|
-|provider_name|`utf8`|
+|provider_name (PK)|`utf8`|
 |provider_type|`utf8`|
```

Comment:
I wonder if `provider_type` should be added to the PK because they could theoretically have the same name

Reply:
No, as we use [`DescribeIdentityProvider`](https://docs.aws.amazon.com/cognito-user-identity-pools/latest/APIReference/API_DescribeIdentityProvider.html) API call that has only `ProviderName` & `UserPoolId` fields